browser-extension/src/components/RegistrationPage.tsx

```diff
@@ -15,22 +15,22 @@ const RegistrationPage: FC = () => {
 
   const register = async () => {
     try {
-      await client.register(email, password, ethAddress);
-      await client.logIn(email, password);
-      await client.createProducer({
+      await apiClient.register(email, password);
+      await apiClient.logIn(email, password);
+      await apiClient.createWallet();
```

Comment:
Create wallet on registration. In the future, I will add the option to enter wallet information on registration if the user already has one.